### ISSUE TITLE
source activate shouldn't completely replace PS1

### ISSUE THREAD

**Issue by kalefranz** [conda/conda#2432]
I don't think we want to completely replace `PS1` with `source activate`.  It violates expectations based on previous conda-env and virtualenv behavior.  It also makes me think I'm in a subprocess--like a python shell or ssh-ing somewhere--where `ctrl-d` is the natural exit.  I've hit `ctrl-d` several times as a reflex testing rc1, when what I really wanted to do was `source deactivate`.

**Comment by kalefranz** [comment]
CC @msarahan

**Comment by msarahan** [comment]
I think what you want is something like https://pypi.python.org/pypi/conda-workon/0.3.0

I pointed this out to @asmeurer a long time ago - and he does not like that workflow.  Since he doesn't, I'm sure others have a like mind.  I think it is important to keep source activate / source deactivate as it is, but add a new command: ```conda enter``` was what I proposed a couple of months ago. @teoliphant liked the idea at the time, but I have not had time to work on it.

This does not preclude a change away from ```source activate``` to ```conda activate``` - I just think we need two activation methods:

  - one that alters your existing shell process
  - one that creates a new shell process
